### PR TITLE
Fix Multiple choices questions

### PR DESCRIPTION
**Pull request by draganakovacevic**
Fix warnings for equivalence

### CODE REVIEW

File: core/src/main/java/eu/execom/testgenerator/core/service/question/QuestionServiceImpl.java [hunk 1]
```diff
@@ -156,10 +157,14 @@ public void remove(Long id) {
   }
 
   private boolean isValid(Question question) {
+    boolean valid = true;
     if (question.isMultipleChoice()) {
-      return question.getAnswers().stream().anyMatch(answer -> answer.isActive() && answer.isCorrect());
+      valid = question.getAnswers().stream().anyMatch(answer -> answer.isActive() && answer.isCorrect());
+      if(!valid) {
+        throw new ValidationException("Question must have a checked correct answer. ");
+      }
```
Comment (stanislavgrujic) [inline core/src/main/java/eu/execom/testgenerator/core/service/question/QuestionServiceImpl.java:165]: This change will prevent saving invalid questions, while the issue description says *"The system should not include such a question when the validity of the formula is checked."*.
Which one is the correct behaviour?

File: core/src/main/java/eu/execom/testgenerator/core/service/question/QuestionServiceImpl.java [hunk 2]
```diff
@@ -156,10 +157,14 @@ public void remove(Long id) {
...
     }
-    return true;
+    return valid;
```
Comment (stanislavgrujic) [inline core/src/main/java/eu/execom/testgenerator/core/service/question/QuestionServiceImpl.java:167]: Why this change when the method always returns *true* and throws an exception in case that it is *false*?

File: core/src/main/java/eu/execom/testgenerator/core/service/question/QuestionServiceImpl.java (hunk 1 above)
Comment (draganakovacevic) [reply core/src/main/java/eu/execom/testgenerator/core/service/question/QuestionServiceImpl.java:165]: Understand, i set that we need correct answer for question to be valid.
